src/main/java/ParkingLot/Admin.java

```diff
@@ -0,0 +1,20 @@
+package ParkingLot;
+
+public class Admin extends Employees {
+
+    public Admin() {
+        standard();
+    }
+
+    public Admin(String name, int workHours) {
+        super(name, workHours);
+        standard();
+    }
+
+    @Override
+    public void standard() {
```

Comment:
naming convention violation. method names should be verbs, and should describe its behavior

Reply:
fixed, changes pushed to mmykytovych_master